content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -264,6 +264,16 @@ For more information, see the [DogStatsD for C# repository][1].
 
 {{< /programming-lang-wrapper >}}
 
+### Run multiple metrics processing pipeline to limit packet drops
+
+If you observe that your DogStatsD server using UDS is dropping packets at a high throughput, configuring it to use more CPU could help processing packets faster, and more of them.
+
+Symptoms are the client-side telemetry indicating that the client is dropping packets in its queue and the DogStatsD server not using more than 2 CPUs/2 cores while the server has more available.
```

Comment:
```suggestion
You may also configure your DogStatsD server if the server does not use more than 2 CPUs or 2 cores even if they are available.
```

Reply:
We're losing the client-side telemetry information here which is important.

What do you think of:

```
You may also configure your DogStatsD server if the client telemetry indicates packet drops and the server does not use more than 2 CPUs or 2 cores even if they are available.
```